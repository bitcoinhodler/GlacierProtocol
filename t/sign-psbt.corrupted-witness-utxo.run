#!/bin/bash

# The PSBT here has been corrupted so the witness_utxo amount does not
# match the non_witness_utxo amount.

$GLACIERSCRIPT --regtest=$1 sign-psbt << INPUT 2>&1
y
y
y
y
y
y
../../t/sign-psbt.corrupted-witness-utxo.psbt
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected glacierscript.py to fail"
    exit 1
fi
