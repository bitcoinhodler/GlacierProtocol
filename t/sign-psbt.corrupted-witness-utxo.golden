Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
ERROR: witness_utxo did not match non_witness_utxo. Exiting...
